packages/mdc-textfield/_mixins.scss

```diff
@@ -261,9 +271,11 @@
   @include mdc-floating-label-float-position($mdc-text-field-outlined-label-position-y);
   @include mdc-floating-label-shake-animation(text-field-outlined);
   @include mdc-text-field-outline-corner-radius_($mdc-text-field-border-radius);
+  @include mdc-states-base-color(transparent);
+  @include mdc-text-field-fill-color(transparent);
 
-  height: 56px;
   border: none;
+  overflow: visible;
```

Comment:
Could you explain what this does in the context of the outlined textfield and why it's necessary to make the content overflow visible?

Reply:
In default, overflow is set to visible. When moving the Box variant to the new default it made overflow to hidden in the mdc-text-field class. It is necessary to set it to visible because when the label floats to the border of the box if it was not visible then the top half of the text would be cut.